Jetcaster/tv-app/src/main/java/com/example/jetcaster/tv/ui/component/EpisodeCard.kt

```diff
@@ -87,7 +78,12 @@ private fun EpisodeThumbnail(
         scale = CardScale.None,
         modifier = modifier,
     ) {
-        AsyncImage(model = playerEpisode.podcastImageUrl, contentDescription = null)
+        AsyncImage(
```

Comment:
Thoughts about using `PodcastImage` in `designsystem`? Would be nice to reuse the same component across form factors. (side-note though: I do plan on update the loading indication from a circular progress to a shimmer effect)

Reply:
I would like to switch to `PodcastImage` from `AsyncImage`, and it is blocked as `PodcastImage` depends on `MaterialTheme` object from `androidx.compose.material3`, which is not a part of `tv-material` library. 

I believe we should remove the dependencies to the mobile specific components from the `designsystem` module, but It would need more discussion. I would defer it to another issue. So, I would keep current implementation in this pull request.